CHANGELOG.md

```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 - Added `anonymize_df` task function to `task_utils.py` to anonymize data in the dataframe in selected columns.
+- Added `GET` connection inside the method `genesys_api_connection`.
+
+### Changed
+- Changed the method to call Genesys API and its name, providing a new paremeter `end_point` to be more general.
+Name changed from `genesys_generate_exports` to `genesys_api_connection`.
+- Changed Genesys source file, deleting methods for testing purposes but never used in production.
```

Comment:
Please correct. -> `Removed methods never used in production: 'methods..' ,.. `

Reply:
the less the better, done!